lib/client/ipam_block_reader_writer.go

```diff
@@ -139,6 +124,20 @@ func (rw blockReaderWriter) claimNewAffineBlock(
 	return nil, noFreeBlocksError("No Free Blocks")
 }
 
+// isIPPoolInRequestedPools checks if the IP Pool that is passed in belongs to the list of IP Pools
+// that should be used for assigning IPs from.
+func isIPPoolInRequestedPools(pool net.IPNet, requestedPools []cnet.IPNet) bool {
```

Comment:
Should mention in the doc that requestedPools may be omitted which means allow all pools.